db/system_keyspace.cc

```diff
@@ -2603,6 +2634,73 @@ future<std::vector<system_keyspace::view_build_progress>> system_keyspace::load_
     });
 }
 
+future<service::vbc::vbc_tasks> system_keyspace::get_view_building_coordinator_tasks() {
+    using namespace service::vbc;
```

Comment:
I object the the name "vbc". It's okay to use short names in local variables, but not in namespace names and class names that will be all over the code and cause readers not to understand what they are reading.